TSQL_Parser/TSQL_Parser/TSQLKeywords.cs

```diff
@@ -149,6 +149,7 @@ public struct TSQLKeywords
 		// public static readonly TSQLKeywords OF = new TSQLKeywords("OF");
 		public static readonly TSQLKeywords OFF = new TSQLKeywords("OFF");
 		public static readonly TSQLKeywords OFFSETS = new TSQLKeywords("OFFSETS");
+		public static readonly TSQLKeywords OFFSET = new TSQLKeywords("OFFSET");

```

Comment:
Nitpicky, but how about adding this to FutureKeywords instead of Keywords? It's not currently a TSQL keyword.

Reply:
If I do that, will it function just the same?  I guess I can run my tests against it after making the change to be sure.

Reply:
It should, as long as you add it to the FutureKeywords list parameter instead in the call to ReadUntilStop.